### PR TITLE
Update docstrings for fftfreq and rfftfreq

### PR DESCRIPTION
**Pull request by jishnub**
This PR updates the docstrings for `fftfreq` and `rfftfreq`, explicitly listing the frequencies returned and adding some examples.

### CODE REVIEW

File: src/definitions.jl [hunk 1]
```diff
@@ -419,8 +419,34 @@ Base.copy(x::Frequencies) = x
...
-bin centers at every sample point. `fs` is the sample rate of the
-input signal.
+bin centers at every sample point. `fs` is the sampling rate of the
+input signal, which is the reciprocal of the sample spacing.
+
+Given a window of length `n` and a sampling rate `fs`, the frequencies returned are 
+
+```julia
+[0, 1, ..., n/2 - 1, -n/2, ..., -1]  * fs/n   # if n is even
+[0, 1, ..., (n-1)/2, -(n-1)/2, ..., -1]  * fs/n  # if n is odd
```
Comment (stevengj) [inline src/definitions.jl:429]: If this is Julia code, you need to make it valid Julia code: `[0:n÷2-1; -n÷2:-1] * fs/n` etcetera.  If you want to use math notation, use a LaTeX block.

Comment (jishnub) [reply src/definitions.jl]: I've fixed this to use valid Julia code, thanks for the suggestion

File: src/definitions.jl [hunk 2]
```diff
@@ -419,8 +419,34 @@ Base.copy(x::Frequencies) = x
...
 `Frequencies` object is an `AbstractVector` containing the frequency
-bin centers at every sample point. `fs` is the sample rate of the
-input signal.
+bin centers at every sample point. `fs` is the sampling rate of the
+input signal, which is the reciprocal of the sample spacing.
+
+Given a window of length `n` and a sampling rate `fs`, the frequencies returned are 
+
+```julia
+[0:n÷2-1; -n÷2:-1]  .* fs/n   # if n is even
```
Comment (stevengj) [inline src/definitions.jl:428]: No need for `.*`

Comment (stevengj) [reply src/definitions.jl:428]: that is, `*` also works here and is simpler

Comment (jishnub) [reply src/definitions.jl]: Ah indeed, I've updated it now. Thanks again!

**Pull request merged**